src/ngx_http_encrypted_session_cipher.c

```diff
@@ -39,7 +39,7 @@ ngx_http_encrypted_session_aes_mac_encrypt(ngx_pool_t *pool, ngx_log_t *log,
         return NGX_ERROR;
     }
 
-    EVP_CIPHER_CTX_init(&ctx);
+    ctx = EVP_CIPHER_CTX_new();
```

Comment:
This call may fail. Better check if `ctx` is NULL?

Reply:
Also, can we reuse this allocated `ctx` data across different invocations? It would be great if we can save the expensive dynamic allocation on such a hot code path.